
# HG changeset patch
# User Victor Stinner <victor.stinner@gmail.com>
# Date 1483653533 -3600
# Node ID fad67c66885f0bd9ebafe2a54f0fa12b5a8fe3bf
# Parent  52d671684342cd2dcc804566d4c755634d3210a8
Issue #27961: Define HAVE_LONG_LONG as 1.

Fix backward compatibility issue, HAVE_LONG_LONG was defined but empty, whereas
it is defined as 1 in Python 3.5.

diff --git a/Include/pyport.h b/Include/pyport.h
--- a/Include/pyport.h
+++ b/Include/pyport.h
@@ -39,7 +39,7 @@ Used in:  Py_SAFE_DOWNCAST
 
 // long long is required. Ensure HAVE_LONG_LONG is defined for compatibility.
 #ifndef HAVE_LONG_LONG
-#define HAVE_LONG_LONG
+#define HAVE_LONG_LONG 1
 #endif
 #ifndef PY_LONG_LONG
 #define PY_LONG_LONG long long

